portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiePreferences.testcase

```diff
@@ -0,0 +1,153 @@
+@component-name = "portal-security"
+definition {
+
+	property custom.properties = "feature.flag.LPS-142518=true";
+	property portal.upstream = "true";
```

Comment:
Don't forget to run cases on release run. **property portal.release = "true";**
Meanwhile need to quarantine those blocked cases on release run after add the release property.

Reply:
Thank you, I added this line.